content/millennium/r4/clinical/diagnostics/questionnaire-response.md

```diff
@@ -50,6 +51,7 @@ URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/Struct
 -----------------|-------------------------------------------------------------------|-----------------------------------------------
  `choice-answer` | [`Boolean`](https://hl7.org/fhir/r4/datatypes.html#boolean)       | Indicates answers come from a list of options.
  `note`          | [`Annotation`](https://hl7.org/fhir/r4/datatypes.html#annotation) | Additional details about a given QuestionnaireResponse group item. Includes author and date/time information.
+ `modifiable`    | [`Boolean`](https://hl7.org/fhir/r4/datatypes.html#boolean)       | The extension is present only if activity modifiable is FALSE
```

Comment:
'Indication of whether data is modifiable or not. Only present when false.'

Reply:
https://github.com/cerner/fhir.cerner.com/pull/697/commits/9c5a0a983df9702bbae651e4be4907edbd35b377#diff-65d395ab3c58d2ca0d3ac40728ec9e34347612b86c616594927d663c0ccb1c31L54